sdk/monitor/monitor-query/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
+## 2.0.0-beta.1 (2023-11-08)
+
+- Added `MetricsBatchClient` to support batch querying metrics.
```

Comment:
The client should be named `MetricsBatchQueryClient`.

Reply:
Done. Will Update in the next commit